packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -131,6 +131,8 @@ export interface ParticipantItemProps {
    * Takes in a unique id value of the element you would like to be read before the ParticipantItem.
    */
   ariaLabelledBy?: string;
+  /** Display border around persona coin */
+  isRaisedHand?: boolean;
```

Comment:
this should be generic, something like you have in the persona coin `isActive`

Reply:
I agree this should still be done, we shouldn't couple a concept to the component that is specific to ACS and Teams interop. Our components are supposed to be separated from anything ACS and Teams related so that Contoso can use them in more specific ways to their needs.